app/src/main/java/com/mobiledevelopmentfoodapp/service/RestaurantService.kt

```diff
@@ -9,7 +9,7 @@ import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
 class RestaurantService {
-    suspend fun fetchFoods(): List<Food>?{
+    internal suspend fun fetchFoods(): List<Food>?{
```

Comment:
added internal visibility modifier for this function per best koltin practices

Reply:
Good.